libs/fetcher.js

```diff
@@ -223,7 +224,7 @@ function executeRequest (request, resolve, reject) {
     var service;
     try {
         service = Fetcher.getService(request.resource);
-        if (!service[op]) {
+        if ([OP_CREATE,OP_READ,OP_UPDATE,OP_DELETE].indexOf(op) < 0 || !service[op]) {
```

Comment:
@lingyan Thanks for your advice. I moved the operation check from ```Request``` constructor to after ```!Fetchr.isRegistered``` block. Please confirm my code.